.gitlab/binary_build/serverless.yml

```diff
@@ -7,7 +7,7 @@ build_serverless-deb_x64:
   before_script:
     - !reference [.retrieve_linux_go_deps]
   script:
-    - cd cmd/serverless && go build -mod=mod -a -v -tags serverless
+    - cd cmd/serverless && GOOS=linux go build -mod=mod -a -v -tags serverless
```

Comment:
I don't understand how this makes a difference (since the job is run in a Debian-based image, it will have `GOOS=linux` by default)

Reply:
Yes this is unrelated, I figured that the build was failing on mac machine because of that : https://github.com/DataDog/zstd/issues/25
I'll close this one.